schemas/public/onpremises-kfd-v1alpha2.json

```
@@ -1808,7 +1860,7 @@
         },
         "baseDomain": {
           "type": "string",
-          "description": "The base domain for the auth module"
+          "description": "CURRENTLY NOT IN USE."
```

This is used, it's for all the auth endpoints, dex, pomerium, gangplank. If you search for `.spec.distribution.modules.auth.baseDomain` in the codebase you can find it's usage

---

I tested this and it was not working (hence the issue [#267](https://github.com/sighupio/fury-distribution/issues/267)). I'll test again.

---

I can't reproduce the problem anymore, probably I broke something on my test env.

```suggestion
          "description": "Base domain for the ingresses created by the Auth module (Gangplank, Pomerium, Dex). Notice that when nginx type is dual, these will use the `external` ingress class."
```

---

LGTM